lib/generators/decidim/templates/Gemfile.erb

```diff
@@ -14,6 +14,9 @@ ruby "<%= RUBY_VERSION %>"
 
 gem "decidim", <%= gem_modifier %>
 
+# Uncomment the following line if you want to use decidim-assemblies plugin
+# gem "decidim-assemblies", <%= gem_modifier %>
```

Comment:
We decided to call it a beta feature for now and make it optional since it's likely that not all installations will want it.